msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -979,64 +988,51 @@ protected void validateAcquireTokenSilentParameters(AcquireTokenSilentParameters
 
     @Override
     public void acquireToken(@NonNull final AcquireTokenParameters acquireTokenParameters) {
-        acquireTokenParameters.setAccountRecord(
-                getAccountRecord(
-                        acquireTokenParameters.getAccount(),
-                        getRequestTenantId(
-                                mPublicClientConfiguration,
-                                acquireTokenParameters
-                        )
-                )
-        );
-
-        validateAcquireTokenParameters(acquireTokenParameters);
+        sBackgroundExecutor.submit(new Runnable() {
```

Comment:
Didn't want to introduce this, but it's required for cases when an IAccount is passed-in and a specific `AccountRecord` needs to be authorized - when broker is used, we have to synthesize the account that is passed across the boundary